frozen_lake.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import sys
+from six import StringIO, b
```

Comment:
According to Mo, avoid statements like `from foo import bar`. It's better to use the fully qualified function name like `foo.bar` so it's easier to read.